draft-iyengar-quic-delayed-ack.md

```diff
@@ -255,8 +255,11 @@ is recorded as the largest seen sequence number. The new Packet Tolerance and
 Update Max Ack Delay values MUST be immediately used for delaying
 acknowledgements; see {{sending}}.
 
-On subsequently received ACK-FREQUENCY frames, the endpoint MUST check if this
-is a more recent frame than any previous ones, as follows:
+On a subsequently received ACK-FREQUENCY frame, the endpoint MUST check if this
+frame is more recent than any previous ones, as follows:
+
+- If the sequence number in the frame is not greater than the largest one seen
+  so far, the endpoint MUST ignore this frame.
```

Comment:
I intentionally removed this, because I think it results in indeterminate behavior.  I think you either need to use packet number or sequence number, not both.

Reply:
Ok, I misread this the first time as packet number for some reason.  I think we need to find a different term besides sequence number at some point.  Also, when is sequence number capitalized?  It seems other fields in the frame always are.

Reply:
I had packet number earlier, but that was mistakenly left over. That should simply have been changed to sequence number, which is what this PR does.

The rule we've used is field names are capitalized, but when talking about the concept, it's not. ("sequence number of the frame" vs "Sequence Number field in the frame"). This is the same as how we did sequence number of NCID frames in the transport draft -- both cases are used in the same manner.